python/katana/plugin.py

```diff
@@ -56,13 +56,13 @@ def load_module(self, fullname):
 
         assert fullname in self._insert_table
 
-        # This will get the module from sys.modules if it is already loaded.
-        module = importlib.import_module(self._insert_table[fullname])
+        aliased_name = self._insert_table[fullname]
         if fullname in sys.modules:
-            raise ImportError(
-                f"module {fullname} already exists while trying to insert "
-                f"{self._insert_table[fullname]} at that name."
-            )
+            module = sys.modules[aliased_name]
+        else:
+            module = importlib.import_module(aliased_name)
```

Comment:
should aliasd_name,fullname be inserted here in _insert_table? (or that is done elsewhere)

Reply:
ok based on that assert above, it is probably inserted somewhere else

Reply:
Hey the table is build in `_register_plugin_loader` and passed to the constructor.